java/ql/src/Frameworks/Spring/XML Configuration Errors/insecureOAuthVersion.ql

```diff
@@ -0,0 +1,44 @@
+/**
+ * @name CVE-2018-1260: Remote Code Execution with spring-security-oauth2
+ * @description A malicious user or attacker can craft an authorization request to the authorization endpoint that can lead to a remote code execution when the resource owner is forwarded to the approval endpoint.
+ * @kind Remote Code Execution
+ */
```

Comment:
You're going to want to look at this (and potentially other example queries) for how to format this comment correctly.

https://help.semmle.com/QL/learn-ql/writing-queries/query-metadata.html

Reply:
Thanks for the link. I'll check it .